### PR TITLE
Updating Sarah's contributor record

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

Updating my contributor record to reflect my current work

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
Looks great Sarah. Would you be able to add a testimonial/impact statement that can inspire people to engage with The Turing Way?
Also, can you update the [contributor.md](https://github.com/sgibson91/the-turing-way/blob/update-bio/contributors.md) file? The contributors record will get automatically updated later today.
I am approving this so you don't have to wait for my second review before you can merge this.

**Review by malvikasharan — APPROVED** [review]
Once this update is moved to contributors.md, happy to have this merged.

**Comment by sgibson91** [comment]
> Would you be able to add a testimonial/impact statement that can inspire people to engage with The Turing Way?

I could update this when I have a bit more writing steam https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/impact_statements/sarah-gibson.md

**Pull request merged**
